### DISCUSSION TITLE
Connect StimulusJS controller to ActiveAdmin resource

### DISCUSSION THREAD

**Discussion by huzaifabutt** [activeadmin/activeadmin#8304] [Help]
Hi Everyone, I've recently moved on rails 7 and new to stimulus as well. for some reason when i connect my stimulus controller to normal index.html.erb file (or any other file outside activeadmin) using "data-controller='home'", it works. but when i apply the same logic in activeadmin resource...the controller just never connects to it...Even tho i see data-controller in the markup but for some reason the controller Stimulus controller just doesnt catch it. do i need to do some configuration for it or what?

**Comment by stuffa** [comment]
I had the same problem several years ago, and again this week
I found the following artical useful - expecially the end - its a little dated (rails 6 ish) but it set me in the right direction
https://www.capsens.eu/en/blog/stimulusjs-on-rails-101-f22a0818dff8
I am using rail 7.2, bootstrap, esbuild
i created a new folder 'app/javascript/admin_controllers' to hold the stimulus controllers i need for AA
I populated it with and index.js file and application.js file, copiled from /app/javascript/controllers, and added the stimulus controllers I wanted
I created a new endpoint '/app/assets/javascript/admin.js'  that contained 'import "./admin_controllers' and nothing else.
Then I modified 'app/assets/javascript/active_admin.js' and added '//= require "admin.js"' at the end
